tests/contrib/langchain/test_langchain_community.py

```diff
@@ -1274,3 +1274,65 @@ def test_faiss_vectorstore_retrieval(langchain_community, langchain_openai, requ
             retriever = faiss.as_retriever()
         with request_vcr.use_cassette("openai_retrieval_embedding.yaml"):
             retriever.invoke("What was the message of the last test query?")
+
+
+@pytest.mark.snapshot(
+    ignores=["meta.langchain.request.tool.description"],
+    token="tests.contrib.langchain.test_langchain_community.test_base_tool_invoke",
+)
+def test_base_tool_invoke(langchain_core, request_vcr):
+    """
+    Test that invoking a tool with langchain will
+    result in a 1-span trace with a tool span.
+    """
+    if langchain_core is None:
+        pytest.skip("langchain-core not installed which is required for this test.")
```

Comment:
I agree but the issue is that this specific dependency isn't needed on all tests and if I understand correctly, fixtures are not available on the level of skipif annotations.
So in order to have it available there i'll need to import it on the whole file instead. Which is not very optimal since it's only used on a couple tests.

Do you know of any workaround for this ?

Reply:
I don't know of a workaround if you want to keep using `langchain_core` as a fixture, no.

My personal taste is that it's more traceable to `import langchain_core` in this module and `skipif` if it's absent, than to go sort out why the `langchain_core` fixture is `None`.

Looks like there are ~10 tests that use the `langchain_core` fixture, so it looks like it's going to get loaded in just about all CI cases anyway?

Anyway, it's a very small nit. :)

Reply:
Yeah i completely understand your point. 
I believe I'll just keep it as it is right now since I haven't authored the file before this PR and wouldn't want to change the existing structure until I speak to the original authors.
Thanks for giving it a look Romain ! 🙇